<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HackTEAMMMM</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>959</width>
          <height>1033</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1005</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>958</width>
          <height>1033</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>940</width>
             <height>338</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>940</width>
             <height>338</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QWidget" name="widget_4" native="true">
              <property name="minimumSize">
               <size>
                <width>600</width>
                <height>250</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>600</width>
                <height>250</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>480</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>480</width>
                   <height>200</height>
                  </size>
                 </property>
                 <widget class="QWidget" name="layoutWidget">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>10</y>
                    <width>422</width>
                    <height>181</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="spacing">
                    <number>26</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout">
                     <property name="spacing">
                      <number>10</number>
                     </property>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_2">
                       <property name="spacing">
                        <number>20</number>
                       </property>
                       <item>
                        <widget class="QLabel" name="label">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="pixmap">
                          <pixmap resource="picture.qrc">:/new/icon/pic/icon/login.png</pixmap>
                         </property>
                         <property name="scaledContents">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit">
                         <property name="minimumSize">
                          <size>
                           <width>350</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>350</width>
                           <height>40</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <property name="spacing">
                        <number>20</number>
                       </property>
                       <item>
                        <widget class="QLabel" name="label_2">
                         <property name="minimumSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="pixmap">
                          <pixmap resource="picture.qrc">:/new/icon/pic/icon/password.png</pixmap>
                         </property>
                         <property name="scaledContents">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_2">
                         <property name="minimumSize">
                          <size>
                           <width>350</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>350</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <property name="spacing">
                      <number>10</number>
                     </property>
                     <property name="leftMargin">
                      <number>70</number>
                     </property>
                     <property name="rightMargin">
                      <number>20</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="pushButton">
                       <property name="minimumSize">
                        <size>
                         <width>160</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>160</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
                       </property>
                       <property name="text">
                        <string>登陆</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_2">
                       <property name="minimumSize">
                        <size>
                         <width>160</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>160</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
                       </property>
                       <property name="text">
                        <string>退出</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>93</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>940</width>
             <height>338</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>287</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>448</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
